analytics/integration_test/src/integration_test.cc

```diff
@@ -341,4 +341,50 @@ TEST_F(FirebaseAnalyticsTest, TestSetConsent) {
   did_test_setconsent_ = true;
 }
 
+// Test that it compiles and runs on all platforms.
+TEST_F(FirebaseAnalyticsTest, TestSetDefaultEventParameters) {
+  LogInfo("Testing SetDefaultEventParameters().");
+
+  // Set some default parameters.
+  std::map<std::string, firebase::Variant> defaults;
+  defaults["default_int"] = 123;
+  defaults["default_string"] = "default_value";
+  firebase::analytics::SetDefaultEventParameters(defaults);
+
+  // Log an event - the defaults should be included automatically by the
+  // underlying SDK if logging immediately after setting is supported.
+  // Verification might need manual checking in the Analytics console or
+  // via platform-specific debug logs if possible.
+  firebase::analytics::LogEvent("event_with_defaults");
+  LogInfo("Logged event_with_defaults");
+
+  // Clear one default parameter by setting it to null.
+  defaults["default_int"] = firebase::Variant::Null();
```

Comment:
Note to self: Android behavior is to use 0 for null, change so that it omits them from the bundle instead.

Or.... why did we assume that setting to null would delete it? It's a regular old map, maybe just delete it from the map directly.

Reply:
(although it is good to pass in a bunch of different types, to ensure type conversion works on each platform)